src/vaults/interfaces/yield-sources.interface.ts

```diff
@@ -1,10 +1,10 @@
-import { ValueSource } from '@badger-dao/sdk';
+import { YieldSource } from '@badger-dao/sdk';
 
 export interface YieldSources {
   apr: number;
-  sources: ValueSource[];
+  sources: YieldSource[];
```

Comment:
in general, we will try to move all the internal plumbings to v3 type data and then convert to older ones on v2 where we need to

Reply:
i like this approach, it sets it up much better to look deprecated and easier to remove later 👍

Reply:
it would be nice, and this might already be covered, to basically run the requests through this commit and an old v2-only commit and verify the outputs are the same